WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.xib

```diff
@@ -29,17 +29,28 @@
                                         <nil key="highlightedColor"/>
                                     </label>
                                     <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="240" verticalHuggingPriority="260" text="#00 First Last" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="0" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="Nzo-QV-IP1">
-                                        <rect key="frame" x="0.0" y="21.5" width="106" height="20.5"/>
-                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
-                                        <nil key="textColor"/>
-                                        <nil key="highlightedColor"/>
-                                    </label>
-                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="260" verticalHuggingPriority="240" verticalCompressionResistancePriority="749" text="Order Status" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="6EE-dX-ERE" customClass="PaddedLabel" customModule="WooCommerce" customModuleProvider="target">
-                                        <rect key="frame" x="0.0" y="48.5" width="97.5" height="20.5"/>
+                                        <rect key="frame" x="0.0" y="20.5" width="106" height="20.5"/>
                                         <fontDescription key="fontDescription" type="system" pointSize="17"/>
                                         <nil key="textColor"/>
                                         <nil key="highlightedColor"/>
                                     </label>
+                                    <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="0Ps-WY-D2t">
+                                        <rect key="frame" x="0.0" y="47" width="229" height="22"/>
+                                        <subviews>
+                                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="260" verticalHuggingPriority="240" verticalCompressionResistancePriority="749" text="Order Status" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="6EE-dX-ERE" customClass="PaddedLabel" customModule="WooCommerce" customModuleProvider="target">
```

Comment:
just wondering, what are the layout/constraint changes in the xib that help fix the issue? I compared the view hierarchy of this branch vs. develop, but not too sure which might have caused the clipped status label before

Reply:
The changes to the xib file are only about nesting the Status label inside a UIView, because the constraints of the stackview or of the other labels compact the width of the status label, which become large like the label above and vice-versa (based on who had more priority). Adding a UIView with the nested status label, fix the problem because the UIView will follow the width of the stackview, but the status label will be resized based on the length of the text.

Reply:
I'm baffled by how placing the label inside a `UIView` fixes this. It sounds like an iOS bug. 🤷‍♂ Thanks for fixing it!